src/backend/base/langflow/api/v1/flows.py

```diff
@@ -132,7 +132,7 @@ def read_flows(
     settings_service: SettingsService = Depends(get_settings_service),
     remove_example_flows: bool = False,
     components_only: bool = False,
-    get_all: bool = False,
+    get_all: bool = True,
```

Comment:
Might be good to leave a note here indicating that 
a) This is required to be `True` for backwards compatibility
b) The release version at which changes should be made here (since we do want to default to `False` at our next  major release)